docs/manual/cli/automator.de.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "contao:automator"
+description: "Allgemeine Aufgaben rund um die Wartung einer Contao-Installation."
+aliases:
+    - /de/cli/automator/
+weight: 15    
+---
+
+
+Dieser Befehl ist eine Schnittstelle zur Contao Klasse `Automator`. Er besteht hauptsächlich aus allgemeinen Aufgaben 
+rund um die Wartung einer Contao-Installation.
+
+```bash
+php vendor/bin/contao-console contao:automator [<task>]
+```
+
+Der Befehl selbst kann ohne Angabe einer Aufgabe ausgeführt werden. Es wird dann nach einer Aufgabe gefragt. 
+Die folgenden Aufgaben sind verfügbar.
+
+| Aufgabe | Beschreibung |
+| --- | --- |
+| `purgeSearchTables`     | Löscht den Suchindex durch löschen der Tabellen `tl_search` und `tl_search_index`. |
+| `purgeUndoTable`        | Entfernt die zwischengespeicherten Rückgängig-Einträge und ermöglicht so die Wiederherstellung gelöschter Einträge. |
```

Comment:
`purgeUndoTable` truncates `tl_undo`. If you purge the undo table you will _not_ be able to restore deleted records anymore.